components/AddedExercisesList/AddedExercise/index.js

```diff
@@ -0,0 +1,18 @@
+import OpenCard from "./openCard";
+import { useState } from "react";
+
+export default function AddedExercise({ exercise }) {
+  const [openCard, setOpenCard] = useState(false);
+
+  function openCardSetter() {
+    setOpenCard(!openCard);
+  }
+  return (
+    <li key={exercise.id} style={{ listStyleType: "none" }}>
+      <button onClick={openCardSetter}>
+        {exercise.name} {String.fromCharCode(openCard === false ? 9660 : 9650)}
+      </button>
+      {openCard === true ? <OpenCard id={exercise.id} /> : <></>}
```

Comment:
When we conditionally render something in react, the general approach seems to be the following. 

```suggestion
      {openCard && <OpenCard id={exercise.id} /> }
```